tests/tensorflow/test_keras_model_export.py

```diff
@@ -262,7 +262,7 @@ def test_signature_and_examples_are_saved_correctly(model, data):
                 if example is None:
                     assert mlflow_model.saved_input_example_info is None
                 else:
-                    assert all((_read_example(mlflow_model, path) == example).all())
+                    np.testing.assert_allclose(_read_example(mlflow_model, path), example)
```

Comment:
is there a precision issue here?

Reply:
No.

I changed the test because we limit the Tensorflow model only accepting TensorSpec input schema, but the original test inferring out a ColSpec schema.